### ISSUE TITLE
Fallout NV: Radio skip song after ~40sec

### ISSUE THREAD

**Issue by t14462** [Frogging-Family/wine-tkg-git#97]
Affected libs: Quartz.dll, mpg123, XACT/FAudio

Tested on clean GOG installation (DRM-Free, without mods, old configs removed)
It is not depend on my pulseaudio settings, with or without pulseaudio-staging patchset
EAX disabled to

[prepare.log](https://github.com/Frogging-Family/wine-tkg-git/files/4994228/prepare.log)
[last_build_config.log](https://github.com/Frogging-Family/wine-tkg-git/files/4994229/last_build_config.log)

**Comment by t14462** [comment]
On wine everything allright with graphics, but totally mess with audio

**Comment by t14462** [comment]
UPSTREAM

Tested on wine-staging from repo - same problem

**Comment by Tk-Glitch** [comment]
And your FAudio libs are built with FFMPEG support? The lib32-FAudio package Arch ships doesn't have FFMPEG support (while the 64-bit package does).

**Comment by t14462** [comment]
Hmm... On Manjaro I can't build aur/lib32-ffmpeg because of invalid PGP, and Frogging FAudio depends on it, so maybe problem here

**Comment by t14462** [comment]
I have installed:
chaotic-aur/  ffmpeg (+32bit)
chaotic-aur/faudio-tkg-git
chaotic-aur/lib32-faudio-tkg-git

But this bug still exist. There is no cure for it, Installations of codecs, quartz.dll, FAudio.dll, XACT does not solve this problem.
As I wrote: It also not depends on pulseaudio settings and pa-staging patchset.

Just insignificant upstream bug.

**Comment by llde** [comment]
Are you sure that on Windows it works fully. FNV (as most of the bethesda games of that generation) is a bugfest.

**Comment by t14462** [comment]
> Are you sure that on Windows it works fully? FNV (as most of the bethesda games of that generation) is a bugfest.

Yes, I have Win10 on second HDD and radio play songs from the beginnin to end

Modding community says: it happens when game "dislike" codec. But installation of quartz.dll and l3codecx does not help

Ofcourse I dont play vanilla FNV. I'm playing FNV+FO3 merge, SFW mod (Russian TTW)

**Comment by Tk-Glitch** [comment]
Recent FAudio updates fixed quite a few bugs. It's worth retesting I suppose.
